media-libs/avidemux-core/avidemux-core-2.6.20.ebuild

```diff
@@ -0,0 +1,113 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit cmake-utils
+
+SLOT="2.6"
+
+DESCRIPTION="Core libraries for a video editor designed for simple cutting, filtering and encoding tasks"
+HOMEPAGE="http://fixounet.free.fr/avidemux"
+
+# Multiple licenses because of all the bundled stuff.
+LICENSE="GPL-1 GPL-2 MIT PSF-2 public-domain"
+IUSE="debug nls nvenc sdl system-ffmpeg vaapi vdpau video_cards_fglrx xv"
+KEYWORDS="~amd64 ~x86"
+
+if [[ ${PV} == *9999* ]] ; then
+	KEYWORDS=""
+	EGIT_REPO_URI="https://github.com/mean00/avidemux2.git"
+	EGIT_CHECKOUT_DIR=${WORKDIR}
+
+	inherit git-r3
+else
+	MY_P="${PN/-core/}_${PV}"
+	SRC_URI="mirror://sourceforge/${MY_PN}/${MY_PN}/${PV}/${MY_P}.tar.gz"
+fi
+
+# Trying to use virtual; ffmpeg misses aac,cpudetection USE flags now though, are they needed?
+DEPEND="
+	!<media-video/avidemux-${PV}:${SLOT}
+	dev-db/sqlite:3
+	sdl? ( media-libs/libsdl:0 )
+	system-ffmpeg? ( >=virtual/ffmpeg-9:0[mp3,theora] )
+	xv? ( x11-libs/libXv:0 )
+	vaapi? ( x11-libs/libva:0 )
+	vdpau? ( x11-libs/libvdpau:0 )
+	video_cards_fglrx? (
+		|| ( >=x11-drivers/ati-drivers-14.12-r3
+			x11-libs/xvba-video:0 )
+		)
+"
+RDEPEND="
+	$DEPEND
+	nls? ( virtual/libintl:0 )
+"
+DEPEND="
+	$DEPEND
+	virtual/pkgconfig
+	nls? ( sys-devel/gettext )
+	!system-ffmpeg? ( dev-lang/yasm[nls=] )
+"
+
+S="${WORKDIR}/${MY_P}"
+BUILD_DIR="${S}/buildCore"
```

Comment:
do you really need this? it's better to not have an explicit `BUILD_DIR` and rather fix the build system instead